pebble_helpers/views/mixins/auth.py

```diff
@@ -45,7 +48,7 @@ def check_login(request, *args, **kwargs):
         """Checks whether the user is logged in and returns the appropriate
         view.
         """
-        view = logged_in if request.user.is_authenticated() else not_logged_in
+        view = logged_in if request.user.is_authenticated else not_logged_in
```

Comment:
Can you try to make this compatible across Django versions please

Reply:
this should be OK as it was transitioned through `CallableBool` in older Django versions